.travis.yml

```diff
@@ -1,7 +1,6 @@
 language: ruby
 rvm:
   - 1.9.3
-  - 2.2.1
-  - 2.2.2
-  - 2.2.3
+  - 2.2.6
+  - 2.4.0
```

Comment:
Yes, I think everything after 2.1 will definitely work, and 2.0 will possibly work.